pennylane_lightning/src/operations.hpp

```diff
@@ -333,7 +335,7 @@ Gate_Xq<2> CRZ(const double& parameter) {
 * @param omega the third rotation angle
 * @return the controlled rotation tensor
 */
-Gate_Xq<2> CRot(const double& phi, const double& theta, const double& omega) {
+inline Gate_Xq<2> CRot(const double& phi, const double& theta, const double& omega) {
```

Comment:
Ah so `inline` directly replaces points in the code with the following, rather than calling the function?

Reply:
Yeah, inlining a function sort of places a request so that the function def is included in every translation unit using it instead of placing a function call and jumping to the function body (mostly recommended for shorter functions).

Also, from [cppref](https://en.cppreference.com/w/cpp/language/inline):
> There may be more than one definition of an inline function or variable (since C++17) in the program as long as each definition appears in a different translation unit and (for non-static inline functions and variables (since C++17)) all definitions are identical. For example, an inline function or an inline variable (since C++17) may be defined in a header file that is #include'd in multiple source files.